session/types.proto

```diff
@@ -85,6 +127,21 @@ message ContainerSessionContext {
   refs.ContainerID container_id = 3 [json_name = "containerID"];
 }
 
+// Lifetime parameters of the token. Field names taken from rfc7519.
+message TokenLifetime {
+  // Expiration epoch, the last epoch when token is valid.
+  // For SessionTokenV2 this is the last valid Unix timestamp.
```

Comment:
1. It's very different in semantics.
2. Compatibility warning fires for the removal of old `TokenLifetime`, so we can keep it as is and add this new one for v2.